src/components/profile/profileContents.tsx

```diff
@@ -52,25 +53,51 @@ const ProfileContents = ({ userData }: ProfileContentsProps) => {
     };
 
     return (
-        <div className="flex flex-row flex-wrap w-full">
-            <div className="flex flex-col lg:w-4/12 mx-auto my-5 px-3">
-                <h3 className="flex items-center justify-center space-x-2 mb-4 w-full">
-                    <span>Ostatnio rozwiązane</span>
-                    <FaCheck />
-                </h3>
-
-                <RecentlySolvedListRenderer solves={userData.solutions} />
+        <div className="flex flex-col lg:pt-0">
+            <div className="bg-white rounded-lg flex flex-col self-stretch shadow my-2">
+                <div
```

Comment:
I think that now it makes sense to extract headers for cards to separate component with possibility to configure icon and text :)

Reply:
Extracted